dasharo-compatibility/usb-hid-and-msc-support.robot

```diff
@@ -59,14 +64,13 @@ USB001.003 USB devices detected by OS (Windows 10)
     ${drives}=    Get Lines Matching Regexp    ${out}    ^OK\\s+DiskDrive\\s+.*$
     Should Contain    ${drives}    ${USB_MODEL}
 
-# TODO:
-# - when we obtain Pi-KVM rest-api
-# USB002.001 USB keyboard detected in FW
-#    [Documentation]    Check whether the external USB keyboard is detected
-#    ...    correctly by the firmware and all basic keys work
-#    ...    according to their labels.
-#    Power On
-#    Enter Tianocore
+USB002.001 USB keyboard detected in FW
+    [Documentation]    Check whether the external USB keyboard is detected
+    ...    correctly by the firmware and all basic keys work
+    ...    according to their labels.
+    Power On
+    Enter Setup Menu Tianocore
+    Navigate Through Some Menus
```

Comment:
I'm not sure if we need such bizzare keyword. Is not entering main menu, and perhaps going into another one, like `Enter Device Manager Submenu`

Reply:
Resolved in [recent commit](https://github.com/Dasharo/open-source-firmware-validation/commit/9a7f15c18a71110e706347fdaa24543ddfc08a90)